config/features/flags.go

```diff
@@ -114,9 +114,9 @@ var (
 		Name:  "enable-vectorized-htr",
 		Usage: "Enables new go sha256 library which utilizes optimized routines for merkle trees",
 	}
-	enableForkChoiceDoublyLinkedTree = &cli.BoolFlag{
-		Name:  "enable-forkchoice-doubly-linked-tree",
```

Comment:
Maybe add `enable-forkchoice-doubly-linked-tree` to deprecate list? Although I don't feel too strongly about it, v3 is suppose to be a breaking release anyway